appveyor.yml

```diff
@@ -30,7 +34,7 @@ environment:
     PlatformToolset: v140
 
 install:
-- ps: if ($env:Platform -like 'MinGW*') { choco install mingw --version $env:PlatformToolset $( if ($env:Platform -like '*32') { Write-Output --forcex86 } ) }
+- ps: if ($env:Platform -like 'MinGW*') { choco install mingw --version $env:PlatformToolset $( if ($env:Platform -like '*32') { Write-Output --forcex86 --params /exception:dwarf } ) }
```

Comment:
Is there a reason you explicitly set the exception model?

Reply:
Yes, because clang uses this exception model (DWARF), so when linking with MinGW's linker it will fail complaining that some functions are missing. So, as far as I know, the Clang + MinGW toolchain combination works only with MinGW DWARF exception model toolchain.